package.json

```diff
@@ -115,13 +113,6 @@
     "rollup-plugin-analyzer": "^4.0.0",
     "typescript": "^5.3.3"
   },
-  "browserslist": [
```

Comment:
By removing this, we accept browserslist's defaults. I don't know what is the best browerslist for a large open source project like single-spa. Here are the browsers currently supported by the default browserslist:

```
~/c/single-spa (update-build|✔) $ pnpx browserslist
and_chr 125
and_ff 126
and_qq 14.9
and_uc 15.5
android 125
chrome 126
chrome 125
chrome 124
chrome 123
chrome 109
edge 125
edge 124
firefox 127
firefox 126
firefox 125
firefox 115
ios_saf 17.5
ios_saf 17.4
ios_saf 16.6-16.7
ios_saf 15.6-15.8
kaios 3.0-3.1
kaios 2.5
op_mini all
op_mob 80
opera 110
opera 109
safari 17.5
safari 17.4
samsung 25
samsung 24
```